index.html

```diff
@@ -0,0 +1,118 @@
+                <ul>
+                    <li><a href="index.html" class="active">Home</a></li>
+                    <li><a href="about.html">About Conference</a></li>
+                    <li><a href="about.html/#past-events">Past Events</a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <div class="mobile-menu-pane">
+        <i class="fa-solid fa-xmark" onclick="removeStuffs()"></i>
+        <ul class="mobile-nav-items">
+            <li><a href="index.html" onclick="removeStuffs()" class="nav-item active">Home</a></li>
+            <li><a href="about.html" onclick="removeStuffs()" class="nav-item">About Conference</a></li>
+            <li><a href="about.html/#past-events" onclick="removeStuffs()" class="nav-item">Past Events</a></li>
+        </ul>
+    </div>
+    <section class="intro container">
+        <i class="fa-solid fa-bars hamburger"></i>
+        <p class="intro-Text">"GDIW 2022"</p>
+        <h1>Glocalizing Ghana's Digital <br> & Innovation Ecosystem: Opportunities <br> for Economic Development</h1>
+        <div class="border-white intro-desc">
+            <p>A trilateral program to celebrate the accomplishments of Ghana's digital innovation ecosystem and build <br> momentum towards a common vision for its future development.</p>
+        </div>
+        <p class="venue"><span class="dates">2022.10.26(WED) ~ 28(FRI)</span> <br> @Accra International Conference Centre</p>
+    </section>
+    <section id="highlights" class="container">
+        <h3 class="highlights-title">Main Program</h3>
+        <hr class="divider">
+        <div class="activity-items">
+            <div class="box-h-item flex-activities">
+                <img src="img/conference.png" alt="">
+                <h3>Panel Discustions</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/workshop.png" alt="">
+                <h3>Workshops</h3>
+                <div>
+                    <p>Extrapreneuring Our Way into Digital Transformation</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/speech-bubble.png" alt="">
+                <h3>Panel Discustions</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/exhibition.png" alt="">
+                <h3>Training Session</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/fireball.png" alt="">
```

Comment:
Kindly make sure that you are adding a meaningful alt attribute to all your images. This is really important for screen readers and other accessibility tools. :nerd_face: :nerd_face: